src/main/java/model/Colors.java

```diff
@@ -0,0 +1,10 @@
+package model;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```
enum should be in singular

Reply:
solve this